tutorials/counter/package-lock.json

```diff
@@ -0,0 +1,4931 @@
+{
```

Comment:
No having a lockfile for JS projects is an absolute nightmare. I can't count the number of times I've ran `npm i` instead of `npm ci` and have the whole thing break.